docs/Commands.md

```diff
@@ -204,3 +204,19 @@ Caching & Hooks documentation is available [here](CachingAndHooks.md).
 ## Styles
 
 Styles documentation is available [here](Styles.md).
+
+## CLI message logging
+
+By default GlueStick will use dark theme when logging messages. If you want to use the light one, either set `GS_LOG_LIGHT` environment variable to `true` or pass `--light` (`-l`) option when running command.
+
+For better CI/CD and other build tools support, GlueStick will switch to _machine friendly output_ mode, so instead of colorful header prefixing message, you will get log matching the following format:
```

Comment:
`thus instead of colorful header prefixing message, you will get logs matching the following format`